smithy-aws-protocol-tests/model/restXml/document-structs.smithy

```diff
@@ -32,6 +32,7 @@ apply SimpleScalarProperties @httpRequestTests([
         body: """
               <SimpleScalarPropertiesInputOutput>
                   <stringValue>string</stringValue>
+                  <emptyStringValue></emptyStringValue>
```

Comment:
These also should probably be isolated with the client only tag

Reply:
updated